backend/utilities/sbom_dependency_migration/sbom_dependency_migration/main.py

```diff
@@ -0,0 +1,73 @@
+import argparse
+import simplejson as json
+
+from artemisdb.artemisdb.models import Scan
+from artemislib.aws import AWSConnect
+from artemislib.consts import SBOM_JSON_S3_KEY
+from artemislib.env import SCAN_DATA_S3_BUCKET, SCAN_DATA_S3_ENDPOINT
+from artemislib.logging import Logger
+from sbom_dependency_migration.__version__ import __version__
+from sbom_report.report import get_report
+
+LOG = Logger(__name__)
+
+
```

Comment:
maybe add a bit more info here on what this migration does (scan DB SBOM dependency data => flat JSON file), excluding match scans. could instead be added to the corresponding README file and just refer to that for more info

Reply:
I know the README is sparse but it does say what this utility does.